test/e2e/specs/site-editor/templates.spec.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Templates', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllTemplates( 'wp_template' );
+	} );
+	test( 'Create a custom template', async ( { admin, page } ) => {
+		const templateName = 'demo';
+		await admin.visitSiteEditor();
+		await page.click( 'role=button[name="Templates"]' );
```

Comment:
Nit: This is fine but we now recommend `getByRole` syntax because that's the documented one. I think either is fine though so feel free to ignore :)!

Reply:
Updated [here](https://github.com/WordPress/gutenberg/pull/50797/commits/aa0aa39cb64c6853d9f65448d0e1487155e48267).